krystex/src/main/java/com/flipkart/krystal/krystex/node/Node.java

```diff
@@ -434,16 +475,18 @@ private Inputs getInputsForResolver(ResolverDefinition resolverDefinition, Reque
         inputsValueCollector.computeIfAbsent(requestId, r -> new LinkedHashMap<>());
     ImmutableSet<String> boundFrom = resolverDefinition.boundFrom();
     Map<String, InputValue<Object>> inputValues = new LinkedHashMap<>();
-    for (String boundFromInput : boundFrom) {
-      InputValue<Object> voe = allInputs.get(boundFromInput);
-      if (voe == null) {
-        inputValues.put(
-            boundFromInput,
-            dependencyValuesCollector
-                .computeIfAbsent(requestId, k -> new LinkedHashMap<>())
-                .get(boundFromInput));
-      } else {
-        inputValues.put(boundFromInput, voe);
+    if (!this.skipLogicRequested.getOrDefault(requestId, false)) {
```

Comment:
This check should not be needed because this method should not be called if skipLogicRequested is true